smsapi/Api/Action/Base.cs

```diff
@@ -42,6 +43,30 @@ public T Execute()
             return response;
         }
 
+        public async Task<T> ExecuteAsync()

```

Comment:
Please add tests that use this

Reply:
I've had a talk with Kuba K. and he mentioned that we should get rid of those `async` calls. Is that still a case?

Reply:
Leave it for now, maybe I'll remove it later when it's fully discussed.